ui/widgets/code_editor.go

```diff
@@ -58,10 +65,10 @@ func NewCodeEditor(code string, _ string, theme *chapartheme.Theme) *CodeEditor
 		CornerRadius: unit.Dp(4),
 	}
 
-	c.editor.Submit = false
-	c.editor.SingleLine = false
+	//	c.editor.Submit = false
+	// 	c.editor.SingleLine = false
```

Comment:
```suggestion
```


Commented lines shouldn't be found in a PR

Do either your forgot to clean, or it shouldn't be commented

Reply:
Hi @ccoVeille thanks for your review, I'm still experimenting with gioview package and that why there where may commented code here.